less/tabs.less

```diff
@@ -137,12 +137,12 @@
 .tabPages {
   background: none;
   display: flex;
-  height: @tabPagesHeight;
-  padding: 7px 0;
   justify-content: center;
+  height: @tabPagesHeight;
+  padding: @tabPagesHeight 0 @navbarMenubarMargin 0;
```

Comment:
- between the bookmark toolbar and the tab pages
- between the tab pages and the tabsbar